src/components/lab/NonGreenhouseGases.js

```diff
@@ -21,20 +20,9 @@ const NonGreenhouseGases = () => {
   const highlightAllSideMenuMolecules = useSelector(
     ({ lab }) => lab.highlightAllSideMenuMolecules,
   );
-  const moleculesOnCanvas = useSelector(({ lab }) => lab.moleculesOnCanvas);
-  const canvasIncomplete = moleculesOnCanvas.some(
-    ({ molecule }) => molecule === '',
-  );
 
   return (
-    <GasesContainer
-      gasContainerLabel={t('Non-greenhouse Gases')}
-      showFillAllButton={
-        selectedMoleculeInSideMenu !== null &&
-        !GREENHOUSE_GASES.includes(selectedMoleculeInSideMenu) &&
-        canvasIncomplete
-      }
-    >
+    <GasesContainer gasContainerLabel={t('Non-greenhouse Gases')}>
```

Comment:
Not sure how it's in the `json`, but I would capitalise G, i.e. `Non-Greenhouse Gases`.